src/OpenRiaServices.Tools.TextTemplate/Test/T4CodeTests.cs

```diff
@@ -70,8 +70,10 @@ public void T4CodeGenBasicTest2()
         public void T4CodeGenBasicTest3()
         {
             Type[] domainServiceTypes = new Type[] {
-                typeof(TestDomainServices.EF.Northwind), 
-                typeof(TestDomainServices.LTS.Northwind)            
+                typeof(TestDomainServices.EF.Northwind),
+#if NETFRAMEWORK
```

Comment:
Maybe we ahold consider a define featre_linq2sql or similar to make it more clear why? (Look at collection view define for client core)

Reply:
added a "has_linq2sql"  define for NETFRAMEWORK build.
The name is not optimal and can be changed in the